src/pusher/Notification.ts

```diff
@@ -1,20 +1,21 @@
-const PushNotifications = require("node-pushnotifications");
 import { Error } from "mongoose";
 import * as winston from "winston";
 import { Promise } from "bluebird";
 import { Config } from "../common/Config";
 import { getValueInEth } from "../common/ValueConverter";
 
+const PushNotifications = require("node-pushnotifications");
+const config = require("config");
+
 export class Notification {
     private push: any;
-    private networkSymbol = process.env.NETWORK_SYMBOL || "ETH";
-
+    private networkSymbol = config.get("Network_symbol") || "ETH";
     private settings = {
         apn: {
             token: {
-                key: Buffer.from(process.env.APN_KEY, "base64").toString(),
-                keyId: process.env.APN_KEYID,
-                teamId: process.env.APN_TEAMID,
+                key: Buffer.from(config.get("Pusher.APN.key"), "base64").toString(),
```

Comment:
let's do everything capitalized? What's the best practice?

Reply:
Based on their examples lowercase is preferred, but I prefer to go with uppercase as I was with process.emv.NODE_ENV for familiarity.